spec/spec_helper.rb

```diff
@@ -6,3 +6,42 @@
   Coveralls::SimpleCov::Formatter
 ]
 Coveralls.wear!
+
+ENV['RACK_ENV'] = 'test'
+
+require File.join(File.dirname(__FILE__), '..', 'app/app.rb')
+
+require 'capybara'
+require 'capybara/rspec'
+require 'rspec'
+require 'database_cleaner'
+
+require_relative 'helpers/session'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.